.buildkite/pipeline.build.yml

```diff
@@ -60,6 +60,9 @@
     - bazel test --config=ci $(./scripts/bazel_export_options)
       --test_tag_filters=doctest,-gpu,-team:core
       python/ray/... doc/...
+    - bazel test --config=ci $(./scripts/bazel_export_options)
+      --test_tag_filters=xcommit,-gpu
+      doc/...
```

Comment:
`:book: Doctest (CPU)` and `:book: Doctest (GPU)` haven't tested Python files like `ray_oom_prevention.py` in the past. They test inline code snippets like `testcode`, and there's typically a separate pipeline step for `doc_code`. For example:

https://github.com/ray-project/ray/blob/83785ab29c65d3ba437356fa8b5c823db7e8067c/.buildkite/pipeline.ml.yml#L85-L95

Is there a step that specifically tests `ray-core/doc_code`, and if not, should we create one?

Reply:
oh good catch, I put this in a wrong job, one sec